stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/LinkAttachmentContent.kt

```diff
@@ -77,13 +82,18 @@ public fun LinkAttachmentContent(
     }
 
     val previewUrl = attachment.titleLink ?: attachment.ogUrl
+    val urlWithScheme = previewUrl?.addSchemeToUrlIfNeeded()
 
     checkNotNull(previewUrl) {
         "Missing preview URL."
     }
 
     val hasImage = attachment.imagePreviewUrl != null
     val painter = rememberImagePainter(data = attachment.imagePreviewUrl)
+    val errorMessage = stringResource(
```

Comment:
I am using the unparsed link here for the toast error message on purpose, since the users will see that one in the UI so it's better if the same one appears in the toast.